packages/react-query/src/__tests__/useSuspenseQueries.test-d.tsx

```diff
@@ -167,4 +167,15 @@ describe('UseSuspenseQueries config object overload', () => {
 
     expectTypeOf(query1Data).toEqualTypeOf<string>()
   })
+
+  it('queryOptions with initialData and skipToken in queryFn should not work on useSuspenseQueries', () => {
+    const query1 = queryOptions({
+      queryKey: ['key1'],
+      queryFn: skipToken,
```

Comment:
would be good to have a test that has a conditional skipToken:

```
queryFn: Math.random() > 0.5 ? skipToken : () => Promise.resolve(5)
```

because that’s a lot closer to a real world use-case